Source/Scene/Cesium3DTileset.js

```diff
@@ -950,21 +954,21 @@ define([
                         // Sort children by distance for (1) request ordering, and (2) early-z
                         children.sort(sortChildrenByDistanceToCamera);
 
-                        // With additive refinement, we only request children that are visible, compared
-                        // to replacement refinement where we need all children.
+                        // With additive refinement, we only request or refine when children are visible
                         for (k = 0; k < childrenLength; ++k) {
                             child = children[k];
-                            // Store the plane mask so that the child can optimize based on its parent's returned mask
-                            child.parentPlaneMask = planeMask;
-
                             // Use parent's geometric error with child's box to see if we already meet the SSE
                             if (getScreenSpaceError(t.geometricError, child, frameState) > maximumScreenSpaceError) {
-                                if (child.contentUnloaded) {
-                                    if (child.visibility(cullingVolume) !== CullingVolume.MASK_OUTSIDE) {
+                                // Store the plane mask so that the child can optimize based on its parent's returned mask
+                                child.parentPlaneMask = planeMask;
+                                child.planeMask = child.visibility(cullingVolume);
```

Comment:
Throughout, should we rename `parentPlaneMask` and `planeMask` to something like `visibilityParentPlaneMask` and `visibilityPlaneMask`?